### PR TITLE
MDEV-14296 wsrep sst common from bindir to libexecdir

### PR DESCRIPTION
**Pull request by grooverdan**
I submit this under the MCA.

### CODE REVIEW

**Comment by svoj** [comment]
Hi @grooverdan, thanks for your contribution!

File: cmake/install_layout.cmake [hunk 1]
```diff
@@ -174,6 +177,7 @@ SET(INSTALL_SCRIPTDIR_DEB               "bin")
 SET(INSTALL_SYSCONF2DIR_DEB             "/etc/mysql/conf.d")
 #
 SET(INSTALL_LIBDIR_DEB                  "lib")
+SET(INSTALL_LIBEXECDIR_DEV              "libexec")
```
Comment (illuusio) [inline cmake/install_layout.cmake:180]: Should be DEB not DEV

**Comment by LinuxJedi** [comment]
I think I'm going to see if I can port this to the latest development branch.

**Comment by LinuxJedi** [comment]
This PR has been ported via cherry-picking to MariaDB 11.2 in #2706. Closing this one as appropriate.

**Pull request closed without merging**